packages/preload/src/index.ts

```diff
@@ -1157,6 +1157,22 @@ export function initExposure(): void {
     },
   );
 
+  contextBridge.exposeInMainWorld('updatePodmanDesktop', async (): Promise<void> => {
+    return ipcInvoke('app:update');
+  });
+
+  contextBridge.exposeInMainWorld('podmanDesktopUpdateAvailable', async (): Promise<boolean> => {
+    return ipcInvoke('app:update-available');
+  });
+
+  contextBridge.exposeInMainWorld(
+    'podmanDesktopGetReleaseNotes',
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
same here why do we have several parameters unused

Reply:
they are all used on the renderer side